F77.pm

```diff
@@ -9,8 +9,11 @@ use File::Which qw(which);
 use List::Util qw(first);
 
 our $VERSION = "1.22";
+our $DEBUG;
 
-warn "\nExtUtils::F77: Version $VERSION\n";
+sub debug { return if !$DEBUG; warn @_ }
+
+debug "\nExtUtils::F77: Version $VERSION\n";
```

Comment:
I'd prefer the version to be always printed, debug or not, due to the nautre of the utility.

Reply:
I have restored the `print` and therefore removed this `debug`, since it's the same information.